### PR TITLE
fixed #76 added RevDoublyLinkedList.c

### LINKED ISSUE #76: Doubly linked list
# Doubly linked-list

Implement a general-purpose doubly linked list with proper documentation and all basic methods stated as under.

- [x] Uses templates/generics or anything equivalent for the language used
- [x] Add element
- [x] delete element
- [x] find element
- [x] length of list
- [x] middle of list
- [x] reverse of list

### PR DESCRIPTION
**Pull request by Code-Recursion**

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@Code-Recursion Provide a proper description, reformat the code with additional comments. Your PR can be merged then

**Comment by YashSaxena9** [comment]
Proper documentation required

File: C++/RevDoublyLinkedList.c [hunk 1]
```diff
@@ -0,0 +1,134 @@
+#include <stdio.h>
```
Comment (YashSaxena9) [inline C++/RevDoublyLinkedList.c:1]: Proper documentation / comments missing for funcitonality description

**Review by YashSaxena9 — CHANGES_REQUESTED** [review]

**Review by YashSaxena9 — APPROVED** [review]
PR accepted for merge :)

**Pull request merged**